libs/platform/src/lib/components/action-bar/action-bar.component.html

```diff
@@ -24,20 +24,18 @@
         <h3 fd-action-bar-title *ngIf="!editing" [attr.title]="title" data-tag="actionbar__title">
             {{ title }}
         </h3>
-        <div fd-form-set *ngIf="editing">
-            <div fd-form-item>
-                <input
-                    #inputTitle
-                    fd-form-control
-                    *ngIf="editing"
-                    class="fd-input edit-actionbar-title"
-                    (focusout)="onFocusOut()"
-                    (keydown.enter)="$event.target.blur()"
-                    [(ngModel)]="title"
-                    aria-required="true"
-                    data-tag="actionbar__input"
-                />
-            </div>
+        <div fd-form-item *ngIf="editing">
```

Comment:
don't think this should be included in the PR

Reply:
This should, because it adapt new forms, which are also on platform.